### PR TITLE
Add support emails and KB articles to readme

### PR DESCRIPTION
**Pull request by nfriedly**
I added several links and both the support@ and mobile-support@ emails to the readme now that the Issues tab is disabled.

I kind of winged it for the copy, so please double-check that what I wrote here makes sense.

### CODE REVIEW

**Comment by nfriedly** [comment]
Also, it looks like I don't even have contributor access to this repo, so I had to fork it to make the PR, and I can't assign it to anyone 🤷

**Comment by nfriedly** [comment]
See https://fullstory.slack.com/archives/C1DAWDPM3/p1641417997449900 for context

**Review by patrick-fs — APPROVED** [review]
LGTM - sorry for the delay

File: README.md [hunk 1]
```diff
@@ -75,3 +75,11 @@ We have a few samples to demonstrate this plugin:
...
 
 Much of the logic for adding the attributes originated in the [transform-react-qa-classes](https://github.com/davesnx/babel-plugin-transform-react-qa-classes/) plugin.
+
+## Getting Help
+
+Please refer to our [Knowledge Base article](https://help.fullstory.com/hc/en-us/articles/360049493054-FullStory-s-Annotate-React-plugin-for-Web-Native) or  contact support@fullstory.com for additional help.
+
+### React Native
+
+Please see our [Getting Started with FullStory React Native Recording](https://help.fullstory.com/hc/en-us/articles/360052419133) guide or email mobile-support@fullstory.com for additional help.
```
Comment (JoshMiers-FS) [inline README.md:85]: Recording -> Capture

Comment (patrick-fs) [inline README.md:85]:
```suggestion
Please see our [Getting Started with FullStory React Native Capture](https://help.fullstory.com/hc/en-us/articles/360052419133) guide or email mobile-support@fullstory.com for additional help.
```

Comment (patrick-fs) [reply README.md]: I took care of this ☝️

**Comment by nfriedly** [comment]
I just realized I put two different emails in here - support@fullstory.com and mobile-support@fullstory.com. I think they should probably both be mobile-support (?)

File: README.md [hunk 2]
```diff
@@ -75,3 +75,11 @@ We have a few samples to demonstrate this plugin:
 - [React native](./samples/react-native-app/)
 
 Much of the logic for adding the attributes originated in the [transform-react-qa-classes](https://github.com/davesnx/babel-plugin-transform-react-qa-classes/) plugin.
+
+## Getting Help
+
+Please refer to our [Knowledge Base article](https://help.fullstory.com/hc/en-us/articles/360049493054-FullStory-s-Annotate-React-plugin-for-Web-Native) or  contact support@fullstory.com for additional help.
```
Comment (JoshMiers-FS) [inline README.md:81]: Ye4ah, this should be `mobile-support@fullstory.com` and also there's a double space at `or  contact`

**Pull request merged**